src/azure-cli/azure/cli/command_modules/keyvault/_transformers.py

```diff
@@ -14,6 +14,13 @@ def _multi_transformers(output, **command_args):
     return _multi_transformers
 
 
+def keep_max_results(output, **command_args):
```

Comment:
It seems that there is no logic handle the --maxresults before ?

Reply:
@arrownj `--maxresults` was directly passed into SDK function, but seems no effect.